sentry_streams/src/python_operator.rs

```diff
@@ -73,15 +73,26 @@ impl PythonAdapter {
                 .unwrap()
                 .as_unbound()
                 .clone_ref(py);
-            let message = Message::new_any_message(
-                RoutedValue {
-                    route: self.route.clone(),
-                    payload: RoutedValuePayload::PyStreamingMessage(payload.into()),
-                },
-                convert_py_committable(py, committable_dict).unwrap(),
-            );
-
-            self.transformed_messages.push_back(message);
+            let bound = payload.clone_ref(py).into_bound(py);
```

Comment:
We discussed about doing a proper integration test as a followup but this is now changing the rust logic and this logic should be covered.

There are two options:
1. Add a test that specifically calls this method 
2. Amend one of the tests that rely on this copy of the Delegate to return a PyWatermark as well. https://github.com/getsentry/streams/blob/main/sentry_streams/src/python_operator.rs#L285-L316